modules/dnn/src/dnn.cpp

```diff
@@ -1255,6 +1255,15 @@ struct Net::Impl
                     if (weightableLayer->_biases)
                         weightableLayer->_biases = convertFp16(weightableLayer->_biases);
                 }
+                else
+                {
+                    for (const auto& weights : {"weights", "biases"})
```

Comment:
Does `HAVE_INF_ENGINE` require C++11 compilation mode?

Reply:
@alalek, Thanks! I added
```cpp
if(NOT HAVE_CXX11)
    ie_fail()
endif()
```
before we try to find Inference Engine.